simplestore/src/test/java/com/uber/simplestore/impl/AtomicFileConcurrentTest.kt

```diff
@@ -41,13 +44,32 @@ class AtomicFileConcurrentTest {
     fun `case 1 when parent folder is present will not trigger FileNotFoundException`() {
         //arrange
         makeParentFolder()
-        val targetFile = File(app().dataDir, "files/simplestore/657b3cd7-f689-451b-aca0-628de60aa234/random_key")
+        val targetFile = createTargetFile()
         //act
         val target = targetFile.outputStream()
         //assert
         assertThat(target).isNotNull()
     }
 
+    @Test(expected = AssertionError::class)
```

Comment:
Pay attention that the failing test has been stably reproduced when it is accessed concurrently.